app/main.py

```diff
@@ -1 +1,13 @@
-# write your code here
+import os
+
+
+def copy_file(command: str) -> None:
```

Comment:
The function `copy_file` does not check if the `cp` command is correct. The task checklist requires to check the `cp` command. You should check if the command starts with `cp` and has exactly three parts.